src/components/content/horizon.tsx

```diff
@@ -10,71 +9,39 @@ export interface HorizonProps {
 
 export function Horizon({ data }: HorizonProps) {
   return (
-    <Wrapper>
+    <aside
+      className={clsx(
+        'sm:flex sm:items-stretch sm:justify-between',
+        'px-side pt-8 pb-6 -ml-2.5'
+      )}
+    >
       {data.map((horizonEntry, index) => {
         return (
-          <Item href={horizonEntry.url} key={index} noExternalIcon path={[]}>
-            <Image alt={horizonEntry.title} src={horizonEntry.imageUrl} />
-            <Headline>{horizonEntry.title}</Headline>
-            <Text>{horizonEntry.text}</Text>
-          </Item>
+          <Link
+            className={clsx(
+              'text-brand text !no-underline box-border',
```

Comment:
`text` is unknown, probably left-over

Can you try `hover:no-underline` instead of `!no-underline`?